pkgs/development/python-modules/m3u8/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, python, fetchFromGitHub, requests, iso8601, bottle, pytest, pytestcov }:
+
+buildPythonPackage rec {
+  pname = "m3u8";
+  version = "0.5.2";
+
+  src = fetchFromGitHub {
+    owner = "globocom";
+    repo = pname;
+    rev = version;
+    sha256 = "0p6wmwv1nfa5pyakq5d55w9v142z5ja3db3s3qr44kx895d9lhng";
+  };
+
+  checkInputs = [ bottle pytest pytestcov ];
+
+  checkPhase = ''
+    ${python.interpreter} -m pytest tests/test_parser.py
+    ${python.interpreter} -m pytest tests/test_model.py
+    ${python.interpreter} -m pytest tests/test_variant_m3u8.py
```

Comment:
this can be simply:
```
  pytest tests/test_{parser,model,variant_m3u8}.py
```

Reply:
done

Reply:
pytest can accept multiple paths, and {...} is bash expansion